app/views/merchant/items/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>Edit <%=link_to @item.name, "/items/#{@item.id}" %></h1>
+
+<center>
+  <%= form_for [:merchant, @item] do |f|  %>
```

Comment:
This form routes to the merchant namespace and looks for the items resources.

Reply:
Thanks for the explanation, this is great to see.